### PR TITLE
[Chore] v18.0-rc1 release 

### PR DESCRIPTION
**Pull request by krendelhoff2**
## Description

This PR introduces a new  release in tezos-packaging.

Follows #

### Changes related to the creation of a release (conditional)

- [ ] I updated release number in [meta.json](/serokell/tezos-packaging/tree/master/meta.json).
- [ ] If the native release version was updated, I reset the `letter_version` in [model.py](/serokell/tezos-packaging/tree/master/docker/package/model.py).

#### In case the new Tezos release provides a new protocol and corresponding testnet

- [ ] I supported new protocol in [protocols.json](/serokell/tezos-packaging/tree/master/protocols.json).
- [ ] I supported new protocol and testnet in native packaging.
- [ ] I supported new protocol and testnet in brew formulas.
- [ ] I added tests for the new protocol and testnet.

#### Update brew bottles and repository mirrors
- [ ] I merged PR that updates bottles hashes (this PR is supposed to be created by the CI).

#### Update documentation

- [ ] I updated [README.md](/serokell/tezos-packaging/tree/master/README.md).
- [ ] I updated [baking doc](/serokell/tezos-packaging/tree/master/docs/baking.md).

### CODE REVIEW

File: scripts/update-tezos.sh [hunk 1]
```diff
@@ -8,14 +8,14 @@
 
 set -e
 
-git config user.name "serokell-bot" # necessary for pushing
-git config user.email "tezos-packaging@serokell.io"
+git config user.name "Savely Krendelhoff" # necessary for pushing
+git config user.email "alicecandyom@proton.me"
```
Comment (pasqu4le) [inline scripts/update-tezos.sh]: This seems like a leftover change that should be removed

Same for the other changes in this file and commit (dd0887e1ba2db35ce28ad04515fdc2d1167f0824)

File: Formula/tezos-accuser-PtMumbai.rb [hunk 2]
```diff
@@ -9,9 +9,9 @@ class << self
...
   homepage "https://gitlab.com/tezos/tezos"
 
-  url "https://gitlab.com/tezos/tezos.git", :tag => "v17.3", :shallow => false
+  url "https://gitlab.com/tezos/tezos.git", :tag => "v18.0-rc1", :shallow => false
 
-  version "v17.3-1"
+  version "v18.0-rc1-1"
 
   build_dependencies = %w[pkg-config coreutils autoconf rsync wget rustup-init cmake]
   build_dependencies.each do |dependency|
```
Comment (pasqu4le) [inline Formula/tezos-accuser-PtMumbai.rb:17]: Now that `master` has new bottles for the latest release, please don't forget to also reset them when rebasing

File: scripts/update-tezos.sh [hunk 3]
```diff
@@ -8,14 +8,14 @@
 
 set -e
 
-git config user.name "Savely Krendelhoff" # necessary for pushing
-git config user.email "alicecandyom@proton.me"
+git config user.name "serokell-bot" # necessary for pushing
+git config user.email "tezos-packaging@serokell.io"
```
Comment (pasqu4le) [inline scripts/update-tezos.sh]: I see now that the changes in this file have been reverted in a later commit (0acb266efc1fe1cd220a95f674dbed7659679782), but they shouldn't be committed at all IMO

File: baking/src/tezos_baking/wizard_structure.py [hunk 4]
```diff
@@ -304,7 +304,6 @@ def url_is_reachable(url):
 networks = {
     "mainnet": "Main Tezos network",
     "ghostnet": "Long running test network, currently using the mumbai Tezos protocol",
```
Comment (pasqu4le) [inline baking/src/tezos_baking/wizard_structure.py]: It seems that you missed this in d20d1aa3589a142f6694e75518348d9971228d98: 
```suggestion
    "ghostnet": "Long running test network, currently using the nairobi Tezos protocol",
```

File: nix/modules/tezos-accuser.nix [hunk 5]
```diff
@@ -8,7 +8,9 @@ with lib;
 let
   octez-accuser-pkgs = {
     "PtNairob" =
-      "${pkgs.octezPackages.octez-baker-PtNairob}/bin/octez-baker-PtNairob";
+      "${pkgs.octezPackages.octez-accuser-PtNairob}/bin/octez-accuser-PtNairob";
+    "Proxford" =
+      "${pkgs.octezPackages.octez-accuser-Proxford}/bin/octez-accuser-Proxford";
```
Comment (pasqu4le) [inline nix/modules/tezos-accuser.nix:13]: This seems an odd change, how come it was a `baker` binary before and now it's an accuser? :thinking:

File: nix/build/hacks.nix [hunk 6]
```diff
@@ -3,19 +3,19 @@
...
-  tezos-protocol-environment = super.tezos-protocol-environment.overrideAttrs (o: {
+  octez-libs = super.octez-libs.overrideAttrs (o: {
     postFixup = ''
       DUMMY_DOCS_DIR="$OCAMLFIND_DESTDIR/../doc/${o.pname}"
       mkdir -p "$DUMMY_DOCS_DIR"
       for doc in "README.md" "CHANGES.rst" "LICENSE"; do
         touch "$DUMMY_DOCS_DIR/$doc"
       done
+
+      DUMMY_ODOC_PAGES_DIR="$DUMMY_DOCS_DIR/odoc-pages"
```
Comment (pasqu4le) [inline nix/build/hacks.nix:14]: It would be nice to have a comment here for future reference (just as there was one before) as to why this is needed

**Review by pasqu4le — CHANGES_REQUESTED** [review]

File: nix/modules/tezos-accuser.nix (hunk 5 above)
Comment (krendelhoff2) [reply nix/modules/tezos-accuser.nix:13]: it should be accuser (see file name), it was baker by mistake

File: .github/workflows/build-bottles.yml [hunk 7]
```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         os: [ { id: macos-12, name: "monterey" } ]
         # we don't bottle meta-formulas that contain only services
-        formula: [tezos-accuser-PtMumbai, tezos-accuser-PtNairob, tezos-admin-client, tezos-baker-PtMumbai, tezos-baker-PtNairob, tezos-client, tezos-codec, tezos-node, tezos-signer, tezos-smart-rollup-client-PtMumbai, tezos-smart-rollup-client-PtNairob, tezos-smart-rollup-node-PtMumbai, tezos-smart-rollup-node-PtNairob]
+        formula: [tezos-accuser-PtNairob, tezos-accuser-Proxford, tezos-admin-client, tezos-baker-PtNairob, tezos-baker-Proxford, tezos-client, tezos-codec, tezos-node, tezos-signer, tezos-smart-rollup-client-PtNairob, tezos-smart-rollup-client-Proxford, tezos-smart-rollup-node-PtNairob, tezos-smart-rollup-node-Proxford]
```
Comment (pasqu4le) [inline .github/workflows/build-bottles.yml:20]: shouldn't the new DAC binaries be added here too?

**Review by pasqu4le — COMMENTED** [review]
A few more comments, but please squash your changes (including those needed for these comments).

File: docker/docker-static-build.sh [hunk 8]
```diff
@@ -8,7 +8,7 @@
 
 set -euo pipefail
 
-binaries=("octez-admin-client" "octez-client" "octez-node" "octez-signer" "octez-codec")
+binaries=("octez-admin-client" "octez-dac-client" "octez-dac-node" "octez-client" "octez-node" "octez-signer" "octez-codec")
```
[the remark this answers is not in the document]
Comment (krendelhoff2) [reply docker/docker-static-build.sh:11]: They are listed here, aren't they?

[the remark this answers is not in the document]
Comment (pasqu4le) [reply docker/docker-static-build.sh:11]: uh, sorry, maybe I missed it. Never mind :+1:

**Review by pasqu4le — APPROVED** [review]

**Pull request merged**